src/context/CardContext.tsx

```
@@ -0,0 +1,46 @@
+import React, { createContext, useContext, useState } from "react"
+
+// Define Cart Context
+const CartContext = createContext<any>(null)
+
+// Custom Hook for easier access
+export const useCart = () => useContext(CartContext)
+
+type Props = React.PropsWithChildren<{}> & {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)